CMakeLists.txt

```diff
@@ -206,6 +206,7 @@ endif()
 
 
 # HIP is always required
+find_package(hiprtc REQUIRED PATHS /opt/rocm)
```

Comment:
Please move this after line 256 and enclose into `if(MIOPEN_USE_HIPRTC) ... endif()`

Reply:
Done

Reply:
Good Catch!

Reply:
[resolved]